src/core/scene/a-scene.js

```diff
@@ -110,10 +113,10 @@ var AScene = module.exports = registerElement('a-scene', {
     enterVR: {
       value: function (event) {
         this.setStereoRenderer();
-        if (isMobile) {
-          setFullscreen(this.canvas);
+        if (window.hasNonPolyfillWebVRSupport) {
+          this.stereoRenderer.requestPresent();
         } else {
-          this.stereoRenderer.setFullScreen(true);
+          setFullscreen(this.canvas);
```

Comment:
I'm not sure anymore here. this is getting hacky. @caseyyee, thoughts? (see [comment above](https://github.com/aframevr/aframe/pull/1065/files#r54713426).)